src/sql/parts/dashboard/containers/dashboardHomeContainer.component.ts

```diff
@@ -0,0 +1,30 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import 'vs/css!./dashboardHomeContainer';
+
+import { Component, forwardRef, Input } from '@angular/core';
+
+import { DashboardWidgetContainer } from 'sql/parts/dashboard/containers/dashboardWidgetContainer.component';
+import { DashboardTab } from 'sql/parts/dashboard/common/interfaces';
+import { WidgetConfig } from 'sql/parts/dashboard/common/dashboardWidget';
+
+@Component({
+	selector: 'dashboard-home-container',
+	providers: [{ provide: DashboardTab, useExisting: forwardRef(() => DashboardHomeContainer) }],
+	template: `
+		<div class="scroll-container" #scrollContainer>
+			<div class="scrollable" #scrollable>
+				<dashboard-widget-wrapper *ngIf="properties" [_config]="properties" style="padding-left: 10px; padding-right: 10px; height: 90px; display: block">
```

Comment:
>dashboard-widget-wrapper [](start = 5, length = 24)

Can this properties be collapsible?

Reply:
I was going to do that in another PR since this already got big